### ISSUE TITLE
PolyatomicEntity <- is a - Macromolecule - looks more like a tautology

### ISSUE THREAD

**Issue by markdoerr** [emmo-repo/EMMO#283]
This class - subclass relationship:
PolyatomicEntity <- is a - Macromolecule

looks more like a tautology. 
If there is no strong motivation to keep it, I would remove the Macromolecule subclass.

**Comment by jesper-friis** [comment]
Thanks. These classes lacks elucidations and should definitely be revised.

**Comment by emanueleghedini** [comment]
I agree that there is a lack of elucidations. However they refer to different concepts. A polyatomic entity is literally an entity made of two or more atoms. A macromolecule is according to IUPAC "a molecule of high relative molecular mass, the structure of which essentially comprises the multiple repetition of units derived, actually or conceptually, from
molecules of low relative molecular mass."

I'll keep this issue open, since chemistry branch will be revised after 1.0.0 release (January 2025).
